tests/kafkatest/version.py

```diff
@@ -61,6 +61,7 @@ def get_version(node=None):
         return DEV_BRANCH
 
 DEV_BRANCH = KafkaVersion("dev")
+DEV_VERSION = KafkaVersion("0.10.2.2-SNAPSHOT")
```

Comment:
Hmm.. this makes me thinking, that are we effectively adding one more step on release process, that the release manager need to modify this value?

Reply:
That is correct. Thinking about it, I am wondering if we could get the value from `gradle.properties' file?

Reply:
I do not know how to do that.. or if it is even possible. I think adding this extra step is not too bad as in `release.py` we already wrap some other places where we need to remove `-SNAPSHOT` anyways, this is just for confirming, that in trunk maybe we'd update `release.py` as well.

Reply:
Ack.